server/services/interventionsService.test.ts

```diff
@@ -1552,17 +1552,17 @@ pactWith({ consumer: 'Interventions UI', provider: 'Interventions Service' }, pr
     const actionPlanAppointments = [
       {
         sessionNumber: 1,
-        appointmentTime: '2021-05-13T12:30:000000Z',
```

Comment:
These are all modifications to lines originally introduced by this PR, right? If so can we please incorporate them into the commits where the lines were introduced?

Reply:
yup